try {
    
    xFunction();

} catch(error) {
    console.error(error);
}
